### DISCUSSION TITLE
Resume failed workflow executions from the next node in the workflow

### DISCUSSION THREAD

**Discussion by rebeccasoren** [apache/dolphinscheduler#12460] [Q&A]
Currently, if a task in a workflow instance goes into overtime, the entire process terminates and is tagged as FAILED, even if there are tasks in the workflow instance that are yet to be executed. Rerunning the workflow instance starts execution from the beginning.

Is it possible to resume workflow execution from the next task in the workflow?

For reference,
Job1 had ran overtime, so it failed and the workflow execution terminated. I want to resume workflow execution from Job2.
<img width="424" alt="2022-10-20 12_49_10-Dolphin Scheduler Admin" src="https://user-images.githubusercontent.com/51917800/196882402-d1255bd5-6735-4b8a-a5c1-7798b4b54246.png">

**Comment by DarkAssassinator** [comment]
hi @rebeccasoren , currect sub task will not run due to depend failed.

**Reply by rebeccasoren** [reply]
In many other job schedulers , there is a provision to mark Job1 as a success or to skip it. This allows the workflow to continue running without stopping for the overtime Job1.

Is there any such arrangement in DolphinScheduler?

**Reply by DarkAssassinator** [reply]
> In many other job schedulers , there is a provision to mark Job1 as a success or to skip it. This allows the workflow to continue running without stopping for the overtime Job1.
> 
> Is there any such arrangement in DolphinScheduler?

hi @rebeccasoren, as i know, some PMC/Commiter is deploying this feature. cc @ruanwenjun @caishunfeng
